plugin/plugin.cpp

```diff
@@ -31,24 +30,6 @@ void QBackendPlugin::registerTypes(const char *uri)
         // its children to the main thread.
         singleConnection->registerTypes(uri);
         singleConnection->moveToThread(QCoreApplication::instance()->thread());
-
-        qmlRegisterSingletonType<QBackendObject>(uri, 1, 0, "Backend",
```

Comment:
We will need something very similar to this code in the Singleton callbacks for custom types